mscore/musescore.cpp

```diff
@@ -3309,6 +3309,7 @@ static void loadScores(const QStringList& argv)
                                     }
 
                               MasterScore* score = mscore->readScore(startScore);
+                              score->setStyle(MScore::defaultStyle());
```

Comment:
I think, not here, but only in the section below, if the name starts with ":/", and the following section, it the initial read failed.  Reasoning - user might have specified a "real" score as his start score, we shouldn't apply the default style to that.

Reply:
Oh I didn't even know you could have another start score. Wow I really tunnel visioned on this once I found the code with the break points. Thank you!